scss/components/_vr-controls.scss

```diff
@@ -0,0 +1,110 @@
+.oo-vr-icon {
+  &--move {
+    position: absolute;
+    height: 33%;
+    width: 33%;
+    cursor: pointer;
+    outline: none;
+    @include vendor-prefixes(border-radius, 3px);
+    &--left {
+      top: 50%;
+      left: 0;
+      bottom: auto;
+      right: auto;
+      @include vendor-prefixes(transform, translateY(-50%));
+      &--touched {
+        background: -moz-linear-gradient(left, rgba(0,0,0,0.6) 0%, rgba(255,255,255,0) 100%);
```

Comment:
Is it possible to use a mixin for the gradient backgrounds?

Reply:
\+